Update README.md to fix font requirements for Arch Linux

pandusonu2 opened the pull request:
The current specifications for installation of Servo in Arch Linux fails to run due to requirement of fonts.

The fix is just to include the required package in the required dependencies list in README.md.

<!-- Please describe your changes on the following line: -->

I added the required font package for Arch Linux in the README file. This is in accordance with the comments on the bug #12107, which deals with similar requirements for Gentoo
---
<!-- Thank you for contributing to Servo! Please replace each `[ ]` by `[X]` when the step is complete, and replace `__` with appropriate data: -->
- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

<!-- Either: -->
- [ ] There are tests for these changes OR
- [X] These changes do not require tests because - The changes are in README file

<!-- Also, please make sure that "Allow edits from maintainers" checkbox is checked, so that we can help you if you get stuck somewhere along the way.-->

<!-- Pull requests that do not address these steps are welcome, but they will require additional verification as part of the review process. -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/servo/servo/19867)
<!-- Reviewable:end -->

highfive commented:
Thanks for the pull request, and welcome! The Servo team is excited to review your changes, and you should hear from @mbrubeck (or someone else) soon.

mbrubeck commented:
@bors-servo r+

Thanks!

bors-servo commented:
:pushpin: Commit 662563b has been approved by `mbrubeck`

<!-- @bors-servo r=mbrubeck 662563b74c6922b22b2cd28c608397e075a9fcec -->

bors-servo commented:
:hourglass: Testing commit 662563b74c6922b22b2cd28c608397e075a9fcec with merge 47b3693d7564b32813746c1dbb6ef29dbb08ffaf...

bors-servo commented:
:sunny: Test successful - [android](http://build.servo.org/builders/android/builds/10530), [arm32](http://build.servo.org/builders/arm32/builds/10773), [arm64](http://build.servo.org/builders/arm64/builds/10802), [linux-dev](http://build.servo.org/builders/linux-dev/builds/10823), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/7689), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/7706), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/10899), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/3297), [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/3320), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/7563), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/7634), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/3394), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/3340), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/6205)
Approved by: mbrubeck
Pushing 47b3693d7564b32813746c1dbb6ef29dbb08ffaf to master...

pandusonu2 committed:
Update README.md to fix font requirements for Arch Linux

The current specifications for installation of Servo in Arch Linux fails to run due to requirement of fonts.

The fix is just to include the required package in the required dependencies list in README.md.
